docs/design/2020-04-17-sql-sequence.md

```diff
@@ -0,0 +1,118 @@
+# TiDB SQL SEQUENCE
+
+- Feature Name: Sequences
```

Comment:
Why adding "Feature Name"? It is not in the template.

Reply:
ok